runtimevar/etcdvar/example_test.go

```diff
@@ -63,13 +52,9 @@ func Example_openVariableFromURL() {
 	// Variables set up elsewhere:
 	ctx := context.Background()
 
-	v, err := runtimevar.OpenVariable(ctx, "etcd://myvarname")
+	v, err := runtimevar.OpenVariable(ctx, "etcd://myvarname?decoder=string")
```

Comment:
The parameter is optional, but defaults to decode to a `[]byte`; since the other examples all have an explicit decoder I thought it was better to be consistent. The documentation for Decoder linked above mentions the default (https://godoc.org/gocloud.dev/runtimevar#DecoderByName).